cms/templates/js/course-outline.underscore

```diff
@@ -200,6 +189,14 @@ if (is_proctored_exam) {
                     </p>
                 </div>
             <% } %>
+            <div class="status-hide-after-due">
+                <p>
+                    <% if (xblockInfo.get('hide_after_due')) { %>
+                        <span class="icon fa fa-eye-slash" aria-hidden="true"></span>
+                        <span class="status-hide-after-due-value"> <%- gettext("Subsection will remain hidden after due date") %> </span>
```

Comment:
Nit: Should "Subsection" be parameterized here to the Assignment type of the subsection?  So it would say "Homework", "Lab", "Exam", etc?